### PR TITLE
feat(taikoon): geoblocking

### PR DESCRIPTION
**Pull request by bearni95**
Using [Vercel's geolocation headers](https://vercel.com/guides/geo-ip-headers-geolocation-vercel-functions) to ban the following countries:
```js
const bannedCountries = [
  'AF', // Afghanistan (AF)
  'BY', // Belarus (BY)
  'MM', // Burma/Myanmar (MM)
  'CF', // Central African Republic (CF)
  'CN', // China (CN)
  'CU', // Cuba (CU)
  'CD', // Democratic Republic of the Congo (CD)
  'ET', // Ethiopia (ET)
  'IR', // Iran (IR)
  'IQ', // Iraq (IQ)
  'LB', // Lebanon (LB)
  'LY', // Libya (LY)
  'ML', // Mali (ML)
  'NI', // Nicaragua (NI)
  'KP', // North Korea (KP)
  'RU', // Russia (RU)
  'SO', // Somalia (SO)
  'SS', // South Sudan (SS)
  'SD', // Sudan (SD)
  'SY', // Syria (SY)
  'VE', // Venezuela (VE)
  'US', // United States (US)
]
```

### CODE REVIEW

File: packages/taikoon-ui/src/lib/connect/web3modal.ts [hunk 1]
```diff
@@ -29,13 +30,13 @@ export const web3modal = createWeb3Modal({
   featuredWalletIds: [],
   allowUnsupportedChain: true,
   excludeWalletIds: ['c57ca95b47569778a828d19178114f4db188b89b763c899ba0be274e97267d96'],
-  chainImages,
+  chainImages: [],
```
Comment (KorbinianK) [inline packages/taikoon-ui/src/lib/connect/web3modal.ts:33]: we should configure chain images

File: packages/taikoon-ui/src/routes/+layout.ts [hunk 2]
```diff
@@ -2,4 +2,4 @@
 // which essentially means you turn your app into an SPA.
 // See https://kit.svelte.dev/docs/page-options#ssr
 
-export const ssr = false;
+//export const ssr = false;
```
Comment (KorbinianK) [inline packages/taikoon-ui/src/routes/+layout.ts:5]:
```suggestion
export const ssr = true;
```

File: packages/taikoon-ui/svelte.config.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import adapter from '@sveltejs/adapter-auto'
```
Comment (KorbinianK) [inline packages/taikoon-ui/svelte.config.js:1]: not necessary, auto contains vercel

File: packages/taikoon-ui/package.json [hunk 4]
```diff
@@ -55,6 +55,8 @@
   "type": "module",
   "dependencies": {
     "@openzeppelin/merkle-tree": "^1.0.6",
+    "@sveltejs/adapter-vercel": "^5.3.0",
```
Comment (KorbinianK) [inline packages/taikoon-ui/package.json:58]: not necessary, svelte auto adapter contains vercel

File: packages/taikoon-ui/src/routes/+layout.ts (hunk 2 above)
Comment (bearni95) [reply packages/taikoon-ui/src/routes/+layout.ts:5]: We need SSR enabled for the geolocation headers from Vercel

File: packages/taikoon-ui/src/lib/connect/web3modal.ts (hunk 1 above)
Comment (bearni95) [reply packages/taikoon-ui/src/lib/connect/web3modal.ts:33]: Disabled for SSR configs, bringing those back!

File: packages/taikoon-ui/src/routes/+layout.ts (hunk 2 above)
Comment (KorbinianK) [reply packages/taikoon-ui/src/routes/+layout.ts:5]: And `ssr = true` does that

**Review by 2manslkh — APPROVED** [review]

File: packages/taikoon-ui/src/routes/blocked/+page.svelte [hunk 5]
```diff
@@ -0,0 +1,4 @@
+<script lang="ts">
+</script>
+
+<h1>Seems that you have been blocked</h1>
```
Comment (KorbinianK) [inline packages/taikoon-ui/src/routes/blocked/+page.svelte:4]: I think we need a more descriptive text here and give the user a reason. Maybe even add a list of blocked countries?

**Review by KorbinianK — APPROVED** [review]

**Pull request merged**